airflow-core/src/airflow/api_fastapi/core_api/routes/ui/grid.py

```diff
@@ -170,7 +170,7 @@ def get_dag_structure(
                     DagRun.id.in_(run_ids),
                     SerializedDagModel.id != latest_serdag.id,
                 )
-            )
+            ),
```

Comment:
Nit: otherwise unchanged file
```suggestion
            )
```

Reply:
aah looks like it is failing check on main: https://github.com/apache/airflow/actions/runs/17947462242/job/51038296078?pr=55991

Reply:
Fixed in https://github.com/apache/airflow/pull/55993